### PR TITLE
update Illustrations

### PR DESCRIPTION
**Pull request by dastfs**
### What's new 
All illustration was updated
Non used illustrations was deleted

### CODE REVIEW

File: src/assets/illustrations/052-bicycle.svg [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-<path fill-rule="evenodd" clip-rule="evenodd" d="M18.7501 46.3895C18.0598 46.3895 17.5001 45.8299 17.5001 45.1395C17.5001 44.4492 18.0598 43.8895 18.7501 43.8895C19.4405 43.8895 20.0001 44.4492 20.0001 45.1395C20.0001 45.8299 19.4405 46.3895 18.7501 46.3895ZM15.5557 45.1395C15.5557 46.9038 16.9859 48.334 18.7501 48.334C20.5144 48.334 21.9446 46.9038 21.9446 45.1395C21.9446 43.3753 20.5144 41.9451 18.7501 41.9451C16.9859 41.9451 15.5557 43.3753 15.5557 45.1395Z" fill="#17E5C3"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M38.75 63.3336C38.75 62.7966 39.1853 62.3613 39.7222 62.3613H66.6667C67.2036 62.3613 67.6389 62.7966 67.6389 63.3336C67.6389 63.8705 67.2036 64.3058 66.6667 64.3058H39.7222C39.1853 64.3058 38.75 63.8705 38.75 63.3336Z" fill="#3E61EE"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M61.25 66.1109C61.25 65.574 61.6853 65.1387 62.2222 65.1387L75.5556 65.1387C76.0925 65.1387 76.5278 65.574 76.5278 66.1109C76.5278 66.6478 76.0925 67.0831 75.5556 67.0831L62.2222 67.0831C61.6853 67.0831 61.25 66.6478 61.25 66.1109Z" fill="#3E61EE"/>
+<svg width="144" height="144" viewBox="0 0 144 144" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (hofmann-felix) [inline src/assets/illustrations/052-bicycle.svg]: Can you please double check that all illustrations have `width` and `height` inside their `svg` tag removed? If it is still set, there may be problems with the `IllustrationLayout` component (s. screenshot)

Bildschirm­foto 2023-03-08 um 13 49 21

**Review by hofmann-felix — CHANGES_REQUESTED** [review]
Thank you for importing the illustrations. I like them much more than the old ones ✨

How do you plan to proceed with the illustrations? Are you planning a follow-up PR in which the new illustrations will be added to the components?

**Comment by dastfs** [comment]
> How do you plan to proceed with the illustrations? Are you planning a follow-up PR in which the new illustrations will be added to the components?

@hofmann-felix  For now I have updated the illustrations in the components as shown in the design and I think as we update the content in the components we will update the illustrations, and we can do it in the same PR.

**Review by hofmann-felix — APPROVED** [review]
Thank you for double checking the issue with width/height and for already adding some illustrations to the page 🙌

Adding the remaining illustrations together with the content sounds like a good plan

**Pull request merged**